xpdacq/Ni24.D

```diff
@@ -0,0 +1,27 @@
+# Calibrant: Nickel (Ni)
```

Comment:
Would it be possible to put this in a subfolder? At some point you might want other calibration standards and you might not want them at the top level folder.